lib/rest_kubernetes.rb

```diff
@@ -5,8 +5,10 @@ module Rest
     module Kubernetes
       extend Helper
 
-      def self.populate(path, base_opts, opts)
-        populate_common("/api/<api_version>", path, base_opts, opts)
+      # the api url can either be /api/<api_version> or /apis/#{type}.k8s.io/<api_version> depending on the operation
+      def self.populate(path, base_opts, opts, type: nil)
+        type_str = type ? "s/#{type}.k8s.io" : ""
```

Comment:
looks like unintended `s` needs to be removed

Reply:
@akostadinov, no, the `s` should be there according to https://docs.okd.io/latest/rest_api/

Reply:
my bad, missed that